dbtmetabase/metabase.py

```diff
@@ -403,14 +436,198 @@ def build_metadata_lookups(
 
         return table_lookup, field_lookup
 
+    def extract_exposures(
+        self,
+        models: List[MetabaseModel],
+        output_path: str = "./",
+        output_name: str = "metabase_exposures",
+        include_personal_collections: bool = True,
+        exclude_collections: Iterable = None,
+        **kwargs,
+    ):
+
+        if exclude_collections is None:
+            exclude_collections = []
+
+        logging.debug("Additional args %s", kwargs)
+
+        refable_models = {node.name: node.ref for node in models}
+
+        self.collections = self.api("get", "/api/collection")
+        self.tables = self.api("get", "/api/table")
+        self.table_map = {table["id"]: table["name"] for table in self.tables}
+
+        duplicate_name_check = []
+        captured_exposures = []
+
+        # {"dataset_query":{"type":"native","native":{"query":"select * from dim_core_site_detail","template-tags":{}},"database":2},"display":"table","visualization_settings":{}}
+
+        for collection in self.collections:
+            if collection["name"] in exclude_collections:
+                continue
+            if not include_personal_collections and collection["name"].endswith(
+                "'s Personal Collection"
```

Comment:
You don't need to parse the name to check that it's a personal collection, `personal_owner_id` will be non-null (or null for non-personal collections).

Reply:
Adjusted to use key, cleaner simpler way to do it for sure since we have it